library/Icinga/Web/Controller/ActionController.php

```diff
@@ -506,7 +506,7 @@ protected function postDispatchXhr()
                 rawurlencode($this->view->title . ' :: ' . $this->view->defaultTitle),
                 true
             );
-        } else {
+        } elseif ($resp->getHeader('X-Icinga-Title') === null) {
```

Comment:
I believe we should always avoid overriding this header. Please encapsulate the entire control structure with this condition. So that even if `$this->view->title` is set the header isn't overridden.